client/my-sites/plugins/plugin-details.jsx

```diff
@@ -0,0 +1,440 @@
+							</div>
+						</div>
+					</div>
+					<div
+						className={ classNames(
+							'plugin-details__layout-col',
+							'plugin-details__layout-col-right',
+							{
+								'no-cta': ! shouldDisplayCTA(
+									selectedSite,
+									props.pluginSlug,
+									isPluginInstalledOnsite
+								),
+							}
+						) }
+					>
+						<div className="plugin-details__header">
+							<div className="plugin-details__price">{ translate( 'Free' ) }</div>
+							<div className="plugin-details__install">
+								<CTA
+									slug={ props.pluginSlug }
+									isPluginInstalledOnsite={ isPluginInstalledOnsite }
+								/>
+							</div>
+							<div className="plugin-details__t-and-c">
+								{ translate(
+									'By installing, you agree to WordPress.com’s Terms of Service and the Third-Party plug-in Terms.'
+								) }
+							</div>
+						</div>
+					</div>
+				</div>
+
+				<div className="plugin-details__sites-list">
+					<SitesList
+						fullPlugin={ fullPlugin }
+						isPluginInstalledOnsite={ isPluginInstalledOnsite }
+						{ ...props }
+					/>
+				</div>
+
+				<div className="plugin-details__layout plugin-details__body">
+					<div className="plugin-details__layout-col plugin-details__layout-col-left">
+						{ fullPlugin.wporg ? (
+							<PluginSections
+								className="plugin-details__plugins-sections"
+								plugin={ fullPlugin }
+								isWpcom={ isWpcom }
+								addBanner
+								removeReadMore
+							/>
+						) : (
+							<PluginSectionsCustom plugin={ fullPlugin } />
+						) }
+					</div>
+					<div className="plugin-details__layout-col plugin-details__layout-col-right">
+						<div className="plugin-details__plugin-details-title">
+							{ translate( 'Plugin details' ) }
+						</div>
+						<div className="plugin-details__plugin-details-content">
+							<div className="plugin-details__downloads">
+								<div className="plugin-details__downloads-text title">
+									{ translate( 'Downloads' ) }
+								</div>
+								<div className="plugin-details__downloads-value value">
+									{ formatNumberCompact( fullPlugin.downloaded, 'en' ) }
+								</div>
+							</div>
+							<div className="plugin-details__tested">
+								<div className="plugin-details__tested-text title">
+									{ translate( 'Tested up to' ) }
+								</div>
+								<div className="plugin-details__tested-value value">{ fullPlugin.version }</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</MainComponent>
+	);
+}
+
+function shouldDisplayCTA( selectedSite, slug, isPluginInstalledOnsite ) {
+	return (
+		isPluginInstalledOnsite === false &&
+		selectedSite &&
+		userCan( 'manage_options', selectedSite ) &&
+		isCompatiblePlugin( slug )
+	);
+}
+
+function CTA( { slug, isPluginInstalledOnsite } ) {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+	const selectedSiteId = useSelector( getSelectedSiteId );
```

Comment:
since we already requested `selectedSite` we can directly use `selectedSite.ID` instead of fetching it again from the state.